pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -295,15 +313,15 @@ func (cgc *containerGC) evictSandboxes() error {
 	}
 
 	for podUID, sandboxes := range evictSandboxes {
-		if cgc.isPodDeleted(podUID) {
-			// Remove all sandboxes if pod has been deleted.
+		if cgc.isPodDeleted(podUID) || len(activeSandboxes[podUID]) > 0 {
```

Comment:
1) Why remove all sandboxes when there is active sandbox?
2) Are you sure `sandboxes` is "evictable sandboxes"?